tests/app/dao/notification_dao/test_notification_dao.py

```diff
@@ -1949,3 +1950,49 @@ def test_dao_get_notifications_by_to_field_orders_by_created_at_desc(sample_temp
     assert len(notifications) == 2
     assert notifications[0].id == notification.id
     assert notifications[1].id == notification_a_minute_ago.id
+
+
+def test_dao_create_notification_email_reply_to_mapping(sample_service, sample_notification):
+
+    create_reply_to_email(sample_service, "test@test.com")
+
+    reply_to_address = dao_get_reply_to_by_service_id(sample_service.id)
```

Comment:
Don't think you need this `dao_get_reply_to_by_service_id` function call, the `create_reply_to_email` also returns the `reply_to_address` object, and then you won't need to index into it when calling `dao_create_notification_email_reply_to_mapping`

Reply:
Yes I spotted that and fixed it in the next branch :( fixed here now